common_test.go

```diff
@@ -186,6 +186,68 @@ func TestAmount__issue202(t *testing.T) {
 	}
 }
 
```

Comment:
In general, I wasn't sure of the testing patterns or tools for for this repository, so I went with a basic table-driven test pattern and no assertion library.  If you'd like, I can show you some nice assertions and mocking libraries that we use (specifically smartystreets/assertions and golang/mock).

Reply:
We use table testing elsewhere in our code. I like it as they're usually readable with lots of test cases.

In previous languages and jobs I've tried to use mocks, but they've always added complexity and failures rather than help.